### PR TITLE
Sort keys

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: electron/js/window-manager.js [hunk 1]
```diff
@@ -17,15 +17,14 @@
...
  */
 
-
-const {BrowserWindow} = require('electron');
+const { BrowserWindow } = require('electron');
 
 let primaryWindowId;
 
+const _getPrimaryWindow = () =>
+  primaryWindowId ? BrowserWindow.fromId(primaryWindowId) : BrowserWindow.getAllWindows()[0];
```
Comment (gregor) [inline electron/js/window-manager.js]: Let's not run prettier on things. This easily fits one line in github views.

File: electron/js/window-manager.js [hunk 2]
```diff
@@ -17,15 +17,14 @@
  *
  */
 
-
-const {BrowserWindow} = require('electron');
+const { BrowserWindow } = require('electron');
```
Comment (gregor) [inline electron/js/window-manager.js]: No spaces...

File: electron/js/window-manager.js [hunk 3]
```diff
@@ -17,15 +17,14 @@
...
 
 let primaryWindowId;
 
+const _getPrimaryWindow = () =>
+  primaryWindowId ? BrowserWindow.fromId(primaryWindowId) : BrowserWindow.getAllWindows()[0];
 
-const _getPrimaryWindow = () => primaryWindowId ? BrowserWindow.fromId(primaryWindowId) : BrowserWindow.getAllWindows()[0];
-
-const _setPrimaryWindowId = (newPrimaryWindowId) => primaryWindowId = newPrimaryWindowId;
+const _setPrimaryWindowId = newPrimaryWindowId => (primaryWindowId = newPrimaryWindowId);
```
Comment (gregor) [inline electron/js/window-manager.js]: No code style changes

File: electron/locale/locale.js [hunk 4]
```diff
@@ -44,27 +43,27 @@ const tr = require('./strings-tr');
...
+  lt: 'Lietuvos',
+  nl: 'Nederlands',
+  pl: 'Polski',
+  pt: 'Português do Brasil',
+  ro: 'Română',
+  ru: 'Русский',
+  sk: 'Slovenčina',
+  sl: 'Slovenščina',
+  tr: 'Türkçe',
+  uk: 'Українська',
```
Comment (gregor) [inline electron/locale/locale.js]: These were sorted by value with English on top on purpose as it defines the language order in menu.

File: electron/locale/locale.js [hunk 5]
```diff
@@ -79,31 +78,33 @@ const getCurrent = () => {
...
-const getText = (string_identifier) => {
+const getText = string_identifier => {
   const strings = eval(getCurrent());
   return strings[string_identifier] || en[string_identifier] || '';
 };
 
-const setLocale = (locale) => {
+const setLocale = locale => {
   current = parseLocale(locale);
   settings.save('locale', current);
```
Comment (gregor) [inline electron/locale/locale.js]: Please no other code style changes...

File: electron/js/window-manager.js (hunk 1 above)
Comment (lipis) [reply electron/js/window-manager.js]: I accidentally formatted it :) yep..

**Pull request merged**